src/popup/locales/en.json

```diff
@@ -129,6 +129,7 @@
       "consensusRequiredHelpMsg": "As the creator of this multisig vault you can determine the authorized signers and set the consensus needed for a transaction confirmation (how many signatures are required to confirm each transaction).",
       "createMultisigVault": "Create Vault",
       "errorDuplicatingSigner": "Cannot add the same address as a signer twice",
+      "errorDuplicateMessage": "Duplicated signer’s account address",
```

Comment:
We already have a wording for such an error (one line above), shall we stick to one of them?